api/gateway/gateway.proto

```diff
@@ -64,7 +64,7 @@ message RxMetadata {
   // Frequency in Hz
   uint64  frequency  = 31;
   // Received signal strength in dBm
-  float   rssi       = 32;
+  float   rssi_channel       = 32;
```

Comment:
As you wrote to me on Slack: "GW v1 was ONLY providing rssiS". Then it doesn't make much sense to rename the old field to rssi_channel, does it?